bytecode/tests/test_bytecode.py

```diff
@@ -253,6 +253,34 @@ def test_to_code(self):
         co = code.to_code(stacksize=42)
         self.assertEqual(co.co_stacksize, 42)
 
+    def test_negative_size_unary(self):
+        code = Bytecode()
+        code.first_lineno = 1
+        code.extend([Instr("UNARY_NOT")])
+        with self.assertRaises(RuntimeError):
+            code.compute_stacksize()
+
+    def test_negative_size_binary(self):
+        code = Bytecode()
+        code.first_lineno = 1
+        code.extend([Instr("LOAD_CONST", 1), Instr("BINARY_ADD")])
+        with self.assertRaises(RuntimeError):
+            code.compute_stacksize()
+
+    def test_negative_size_call(self):
+        code = Bytecode()
+        code.first_lineno = 1
+        code.extend([Instr("CALL_FUNCTION", 0)])
+        with self.assertRaises(RuntimeError):
+            code.compute_stacksize()
+
+    def test_negative_size_build(self):
```

Comment:
I would like to see some extra tests in particular for corner cases like DUP_TOP.